packages/@react-spectrum/radio/chromatic/Radio.chromatic.tsx

```diff
@@ -60,6 +60,10 @@ storiesOf('RadioGroup', module)
   .add(
     'long radio label',
     () => renderLongLabel({value: 'dragons'})
+  )
+  .add(
```

Comment:
Is this going to be removed too?

Reply:
That one is ok since it's in RadioGroup. We were just missing CheckboxGroup